Mastodon/Scene/HomeTimeline/HomeTimelineViewController.swift

```diff
@@ -200,6 +200,12 @@ extension HomeTimelineViewController {
             }
             .store(in: &disposeBag)
         
+        context.publisherService.statusPublishResult.sink { result in
+            if case let Result.success(res) = result, case StatusPublishResult.edit = res {
```

Comment:
I've never seen a nested `if case let, case`, but it totally makes sense 👍

Reply:
I think this could also be 

```suggestion
            if case .success(.edit) = result {
```

(but the 2 should be equivalent)